src/app/features/collection/components/cards-list/cards-list.component.html

```diff
@@ -1,3 +1,14 @@
+@if (canMarkAllAsCollected()) {
```

Comment:
cards list and chips list are two different ways to display same entities so the button should work for both of them too. Based on this it looks like we have to move this button at the collection page level, to cover both views

Reply:
we also should not display that button if user in search mode (use searchbar), because in that case search results will be displayed instead of collection cards

Reply:
moved the button to the collection page level